server.js

```diff
@@ -1,11 +1,135 @@
 const express = require('express')
 const app = express()
+const cors = require('cors')
+const bodyParser = require('body-parser')
+const { Dvr } = require('@mui/icons-material')
 const port = 5000
 
+const sqlite3 = require('sqlite3').verbose();
+
+const db = new sqlite3.Database('./shopListDB.db', sqlite3.OPEN_READWRITE, (err) => {
+  if(err) 
+    return console.error(err.message);
+
+  console.log('connection with database successful');
+});
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+app.use(cors({
+  origin: '*',
+  credentials: true
+}));
+
 app.get('/', (req, res) => {
-  res.send('Hello World!')
+  res.send("HELLO WORLD");
+})
+
+app.get('/createlist', (req, res) => {
+  db.all("SELECT * FROM PRODUCTS ORDER BY name", [], (err, products) => {
+    if(err) return console.error(err.message);
+      
+    products.forEach(prod => {
+      console.log(prod);
+    })
+    res.send(products);
+  })
+})
+
+app.get('/showlists', (req, res) => {
+  db.all("SELECT Date FROM SHOPPING_LISTS WHERE Status = 0", [], (err, dates) => {
+    if(err) return console.error(err.message);
+
+    dates.forEach(date => {
+      console.log(date);
+    })
+    res.send(dates);
+  })
+})
+
+app.post('/additem', (req, res) => {
+  db.run("INSERT INTO PRODUCTS(name, department, units) VALUES (?, ?, ?)",
+    [req.body.product, req.body.department, req.body.packeging]), (err) => {
+      if(err) {
+        res.send({inserted: false})
+        return console.error(err.message);
+      };
+    }
+    res.send({inserted: true})
+})
+
+app.get('/closelist', (req, res) => {
+  res.send("CLOSE LIST")
```

Comment:
?

Reply:
Indication that server connected on /closelist

Reply:
I still don't get why do you need `/closelist`